pkg/chartutil/values_test.go

```diff
@@ -447,10 +447,12 @@ chapter:
 	if _, err := d.PathValue("chapter.doesntexist.one"); err == nil {
 		t.Errorf("Non-existent key in middle of path should return error: %s\n%v", err, d)
 	}
+	if _, err := d.PathValue(""); err == nil {
+		t.Errorf("Empty path should return error: %s\n%v", err, d)
```

Comment:
I would replace the statement with the following:
```go
t.Error("Asking for the value from an empty path should yield an error")
```
I didn't punctuate the sentence for consistency with the rest of the failure messages in this file.